first_timers/first_timers.py

```diff
@@ -22,15 +26,17 @@ def humanize_url(api_url):
 
 def get_first_timer_issues():
     """Fetches the first page of issues with the label first-timers-label which are still open."""
-    res = requests.get(query)
-    if res.status_code == 403:
-        warnings.warn('Rate limit reached')
-        return []
-    elif res.ok:
-        return res.json()['items']
-    else:
-        raise RuntimeError('Could not handle response: ' + str(res) + ' from the API.')
-
+    items=[]
+    for query in queries:
+        res = requests.get(query)
+        if res.status_code == 403:
+            warnings.warn('Rate limit reached')
+            return []
```

Comment:
I agree that it would be safe, and both more elegant as well as purposeful. I had simply left it the way it was, but as this is inside a loop now it is more sensible to change it to make use of previous requests. Thanks!